### PR TITLE
{bio}[foss/2017a] CPB v11-4-2011

### PR DESCRIPTION
**Pull request by timeu**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by timeu** [comment]
For some reason I can't add the patch with --update-pr. I get following error: 

```
easybuild@login1:~/gmi_easyconfigs> eb --github-user timeu --update-pr 5869 --pr-commit-msg "Added Makefile to root directory to trigger sub Makefiles" CPB-11-4-2011-foss-2017a-Python-2.7.13.eb CPB-11-4-2011_makefile.patch 
== temporary log file in case of crash /tmp/eb-lFE1v6/easybuild-bLqGz6.log
Please enter password for encrypted keyring: 
== Determined branch name corresponding to easybuilders/easybuild-easyconfigs PR #5869: 20180221133849_new_pr_CPB1142011
== copying /net/gmi.oeaw.ac.at/software/mendel/easybuild-home/easybuild-clones/easybuild-easyconfigs...
== fetching branch '20180221133849_new_pr_CPB1142011' from https://github.com/Gregor-Mendel-Institute/easybuild-easyconfigs.git...
== copying easyconfigs to /tmp/eb-lFE1v6/git-working-dirRmTPlW/easybuild-easyconfigs...
== determining software names for patch files...
Matching easyconfig for CPB-11-4-2011_makefile.patch not found on the first try: scanning all easyconfigs to determine where patch file belongs (this may take a while)...
1718 of 1718 easyconfigs checked
ERROR: Failed to determine software name to which patch file CPB-11-4-2011_makefile.patch relates
```

Could it be that this is due to the explicit patch level ? 
I will add the commit manually

File: easybuild/easyconfigs/c/CPB/CPB-11-4-2011-foss-2017a-Python-2.7.13.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+homepage = 'http://bmi.osu.edu/hpc/software/cpb/index.html'
+description = """CPB is a novel two-step Pearson correlation based biclustering approach to mine genes that are co-regulated with 
+ a given reference gene in order to discover genes that function in a common biological process. 
+ In the first step, the algorithm identifies subsets of genes with high correlation, reducing false negatives 
+ with a nonparametric filtering scheme. 
+ In the second step, biclusters from multiple datasets are used to extract and rank gene correlation information."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = ['http://bmi.osu.edu/hpc/software/cpb/']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/c/CPB/CPB-11-4-2011-foss-2017a-Python-2.7.13.eb]: no longer available at this url?

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/c5050e9178d0f32c92d4994d8923d33b for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/623a2abfd934e50736656afa886049b6 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @timeu!

**Pull request merged**